gpkit/constraints/linked.py

```diff
@@ -4,6 +4,11 @@
 from .. import SignomialsEnabled
 
 
+class SubstitutionConflictError(Exception):
+    "Raised when linked variables have differing values."
+    pass
```

Comment:
maybe just `SubstitutionError`?

Or, if this is not referring to gpkit.substitution, then `LinkingError`?